SU-SWS/react_paragraphs
# D8CORE-2329 Fix CKEditor widget when a user deletes all text

# READY FOR REVIEW

# Summary
- Corrected the logic when a user deletes all contents of the wysiwyg.
- Eliminated console errors when deleting all contents.

# Need Review By (Date)
- 7/8

# Urgency
- High

# Steps to Test
1. Checkout this branch
1. clear browser caches
1. create a basic page with a card on it and fill the card description
1. save the page.
1. see the text that was entered in the card description
1. edit the page and delete all text from the card description. Don't enter any other key other than just deleting it.
1. save the page.
1. validate the text is no longer there.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
